File: php/examples/get_company.php
<?php
require_once 'base_request.php';

// Example of GET /companies/company_id

$debug = true;
$company_id   = $_ENV['COMPANY_ID'];
$request_data = '';
$request_params = [];
// TODO: how Guzzle sends array query params is received by Ruby as indifferent access Hash
// $request_params = ['names' => ['Test Company 2']];

send_request('GET', '/companies/' . $company_id, $request_params, $request_data);

File: csharp/examples/GetCompany.cs
using System;

namespace Examples.examples
{
    // Example of GET /companies/:company_id

    class GetCompany : BaseRequest
    {
        private string _companyId;

        public GetCompany() : base()
        {
            dotenv.net.DotEnv.Load();
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            Console.WriteLine($"GetCompany: company_id={_companyId}");
        }

        public override string SendRequest()
        {
            return GetRequest($"companies/{_companyId}");
        }
    }
}
